securitas/tests/unit/conftest.py

```diff
@@ -83,6 +83,12 @@ def dummy_user(make_user):
     yield
 
 
+@pytest.fixture
+def cleanup_dummy_user():
+    yield
+    ipa_admin.user_del('dummy')
```

Comment:
Same comment as on the other PR: I'm not sure it's a good idea to move fixtures to the shared module if they are not used by more than one test module. I'm afraid it may end up cluttering the `conftest.py` file.